cve_bin_tool/cvedb.py

```diff
@@ -862,9 +883,13 @@ def dict_factory(self, cursor, row):
 
     def get_all_records_in_table(self, table_name):
         """Return JSON of all records in a database table."""
+        # Use the validation function to check the table name
+        if not self.is_valid_table_name(table_name):
+            raise ValueError(f"Invalid table name: {table_name}")
         cursor = self.db_open_and_get_cursor()
         cursor.row_factory = self.dict_factory
-        cursor.execute(f"SELECT * FROM '{table_name}' ")  # nosec
+        # Use parameterized query to prevent SQL injection
+        cursor.execute("SELECT * FROM ? ", (table_name,))
```

Comment:
```suggestion
        # Can't use parameterized query because they don't work on table names
        cursor.execute(f"SELECT * FROM '{table_name}' ")  # nosec
```

Reply:
I think this may be the problem.